runtime/doc/syntax.txt

```diff
@@ -4864,7 +4864,7 @@ SignColumn	column where |signs| are displayed
 							*hl-IncSearch*
 IncSearch	'incsearch' highlighting; also used for the text replaced with
 		":s///c"
-							*hl-Substitute*
+						 {Nvim} *hl-Substitute*
```

Comment:
I was thinking we should not do this anymore (and remove the existing cases). Because `vim_diff` is comprehensive. And we should just remove the old cases of this.